posthog/warehouse/models/datawarehouse_saved_query.py

```diff
@@ -47,6 +57,13 @@ class DataWarehouseSavedQuery(CreatedMetaFields, UUIDModel, DeletedMetaFields):
     )
     external_tables = models.JSONField(default=list, null=True, blank=True, help_text="List of all external tables")
     query = models.JSONField(default=dict, null=True, blank=True, help_text="HogQL query")
+    last_run_status = models.CharField(
+        null=True, choices=Status.choices, max_length=64, help_text="The status of when this SavedQuery last ran."
+    )
+    last_ran_at = models.DateTimeField(
+        null=True,
+        help_text="The timestamp at which this SavedQuery was last ran.",
+    )
```

Comment:
`last_run_` and `last_ran_` - consistency here would be good (if we're keeping the `last_run` prefix. Maybe we just want `model_updated_at` field instead?

Reply:
I think it's useful to have a timestamp specifically to know when was the model last materialized, as that will determine freshness of the data. `*_updated_at` could be confused with updates to the model that do not affect data freshness.